### DISCUSSION TITLE
doubts when using min.io in self hosting

### DISCUSSION THREAD

**Discussion by jongomes** [outline/outline#2411] [Self Hosting]
I need documentation using min.io, it is not able to connect, via browser interface works sending the photo

**Comment by jongomes** [comment]
similar https://wiki.generaloutline.com/share/125de1cc-9ff6-424b-8415-0d58c809a40f

**Comment by almereyda** [comment]
https://github.com/outline/outline/discussions/2466 suggests the connection needs to be via HTTPS.

https://github.com/outline/outline/discussions/1397 also describes a problem with image uploading, and solves this by adding specific CORS headers.
